README.md

```diff
@@ -850,7 +850,16 @@ export const primaryColorVar = createVar();
 export const secondaryColorVar = createVar();
 
 export const exampleStyle = style({
-  color: fallbackVar(primaryColorVar, secondaryColorVar, 'blue');
+  color: fallbackVar(primaryColorVar, secondaryColorVar, 'blue'),
+});
+```
+
+For fallback styles you may simply pass an array of properties instead of a single prop.
```

Comment:
Feel like this part should be part of the the style API section. Maybe just before the "Multiple styles can be composed into a single rule by providing an array of styles" section?

Reply:
Yup! No prob, doable!

Before this one
![image](https://user-images.githubusercontent.com/10671879/150686031-56c7dfda-68e3-4e4e-868f-6cd5a06361f6.png)

or this?
![image](https://user-images.githubusercontent.com/10671879/150686044-c3ef54b3-4a50-4ea1-a588-4b6c16a5bfda.png)

Reply:
@mattcompiles @goodwin64 👀

Reply:
Sorry I missed this. I think the former option would be fine.